datadog/resource_datadog_dashboard.go

```diff
@@ -7613,6 +7795,343 @@ func buildDatadogTraceServiceDefinition(terraformDefinition map[string]interface
 	return datadogDefinition
 }
 
+//
+// Split Graph Definition helpers
+//
+
+func getSplitGraphDefinitionSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"source_widget_definition": {
+			Description: "The original widget we are splitting on.",
+			Type:        schema.TypeList,
+			Required:    true,
+			Elem: &schema.Resource{
+				Schema: getSplitGraphSourceWidgetSchema(),
+			},
+		},
+		"split_config": {
+			Description: "Encapsulates all user choices about how to split a graph.",
+			Type:        schema.TypeList,
+			Required:    true,
+			Elem: &schema.Resource{
+				Schema: getSplitConfigSchema(),
+			},
+		},
+		"size": {
+			Description: "Size of the individual graphs in the split.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"has_uniform_y_axes": {
+			Description: "Normalize y axes across graphs.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+		},
+		"title": {
+			Description: "The title of the widget.",
+			Type:        schema.TypeString,
+			Optional:    true,
+		},
+		"live_span": getWidgetLiveSpanSchema(),
+	}
+}
+
+func getSplitConfigSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"split_dimensions": getSplitDimensionSchema(),
+		"limit": {
+			Description: "Maximum number of graphs to display in the widget.",
+			Type:        schema.TypeInt,
+			Optional:    true,
+		},
+		"sort":          getSplitSortSchema(),
+		"static_splits": getStaticSplitsSchema(),
+	}
+}
+
+func getSplitDimensionSchema() *schema.Schema {
+	return &schema.Schema{
+		Description: "The property by which the graph splits",
+		Type:        schema.TypeList,
+		Required:    true,
+		MinItems:    1,
+		MaxItems:    1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"one_graph_per": {
+					Description: "The system interprets this attribute differently depending on the data source of the query being split. For metrics, it's a tag. For the events platform, it's an attribute or tag.",
+					Type:        schema.TypeString,
+					Required:    true,
+				},
+			},
+		},
+	}
+}
+func getSplitSortSchema() *schema.Schema {
+	return &schema.Schema{
+		Description: "Controls the order in which graphs appear in the split.",
+		Type:        schema.TypeList,
+		Required:    true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"compute": getSplitSortComputeSchema(),
+				"order": {
+					Description:      "Widget sorting methods.",
+					Type:             schema.TypeString,
+					Required:         true,
+					ValidateDiagFunc: validators.ValidateEnumValue(datadogV1.NewWidgetSortFromValue),
+				},
+			},
+		},
+	}
+}
+
+func getSplitSortComputeSchema() *schema.Schema {
+	return &schema.Schema{
+		Description: "Defines the metric and aggregation used as the sort value",
+		Type:        schema.TypeList,
+		Optional:    true,
```

Comment:
```suggestion
		Optional:    true,
		MaxItems:    1,
```
similar to above comments, but no min requirement because its an optional field.